### PR TITLE
Mirrors volume icons in RTL

### PR DESCRIPTION
**Pull request by fstanis**

### CODE REVIEW

File: audio-ui/src/main/java/com/google/android/horologist/audio/ui/VolumeScreen.kt [hunk 1]
```diff
@@ -199,7 +202,12 @@ public object VolumeScreenDefaults {
     public fun IncreaseIcon() {
         Icon(
```
Comment (yschimke) [inline audio-ui/src/main/java/com/google/android/horologist/audio/ui/VolumeScreen.kt:203]: What about a StandardIcon in base-ui that has this as a Flag?

Comment (yschimke) [reply audio-ui/src/main/java/com/google/android/horologist/audio/ui/VolumeScreen.kt:203]: How many icons does this likely affect, why just volume ones?

Comment (fstanis) [reply audio-ui/src/main/java/com/google/android/horologist/audio/ui/VolumeScreen.kt]: Done. Good shout, it did increase min SDK to 26 though.

File: audio-ui/src/main/java/com/google/android/horologist/audio/ui/components/actions/SettingsButton.kt [hunk 2]
```diff
@@ -59,13 +61,14 @@ public fun SettingsButton(
...
     ) {
-        Icon(
+        StandardIcon(
             imageVector = imageVector,
             contentDescription = contentDescription,
             modifier = Modifier
                 .size(iconSize)
                 .align(Alignment.Center)
-                .semantics { iconImageVector = imageVector }
+                .semantics { iconImageVector = imageVector },
```
Comment (yschimke) [inline audio-ui/src/main/java/com/google/android/horologist/audio/ui/components/actions/SettingsButton.kt:70]: nit: put into StandardIcon?

**Review by yschimke — APPROVED** [review]

**Pull request merged**